Options.py

```diff
@@ -917,6 +923,54 @@ def from_any(cls, data: typing.Any):
             return cls(data)
         return cls.from_text(str(data))
 
+    def verify(self, world: typing.Type[World], player_name: str, plando_options: PlandoOptions) -> None:
+        if self.random_str and not self.value:
+            choice_list = sorted(self.valid_keys)
+            if self.verify_item_name:
+                choice_list.extend(sorted(world.item_names))
+                if self.convert_name_groups:
+                    choice_list.extend(sorted(world.item_name_groups.keys()))
+            if self.verify_location_name:
+                choice_list.extend(sorted(world.location_names))
+                if self.convert_name_groups:
+                    choice_list.extend(sorted(world.location_name_groups.keys()))
+            if self.random_str == "random":
+                choice_count = random.randint(0, len(choice_list))
+            elif self.random_str == "random-low":
+                choice_count = int(round(random.triangular(0, len(choice_list), 0)))
+            elif self.random_str == "random-high":
+                choice_count = int(round(random.triangular(0, len(choice_list), len(choice_list))))
+            elif self.random_str == "random-middle":
+                choice_count = int(round(random.triangular(0, len(choice_list))))
+            elif self.random_str.startswith("random-range-"):
+                textsplit = self.random_str.split("-")
+                try:
+                    random_range = [int(textsplit[len(textsplit) - 2]), int(textsplit[len(textsplit) - 1])]
```

Comment:
you can just do this right?
```suggestion
                    random_range = [int(textsplit[-2]), int(textsplit[-1])]
```